File: raeunlee/Week1/Day1/PGS_같은숫자는싫어.py
#풀이시간: 5분, 검색안함

def solution(arr):
    answer = []    
    tmp = arr[0]
    answer.append(tmp)
    for i in range(1, len(arr)):
        if arr[i] == arr[i-1]:
            continue
        else:
            answer.append(arr[i])
    return answer

File: hjkim/Week1/01.02/PGS_같은 숫자는 싫어.java
import java.util.*;

public class Solution {
    public ArrayList<Integer> solution(int[] arr) {
        ArrayList<Integer> answer = new ArrayList<>();
        answer.add(arr[0]);
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] != arr[i - 1]) {
                answer.add(arr[i]);
            }
        }
        return answer;
    }
}